project-euler/task1/README.md

```diff
@@ -0,0 +1 @@
+Project Euler
```

Comment:
Markdown syntax to highlight the title for example you could use
```suggestion
# Project Euler
```

Reply:
Highlighted in the README.md file.